demos/tabs.html

```diff
@@ -0,0 +1,549 @@
+        margin-bottom: 32px;
+      }
+
+      #dynamic-demo-toolbar h1 {
+        margin-left: 64px;
+      }
+
+      section + section {
+        margin-top: 40px;
+        padding-top: 40px;
+        border-top: 1px solid lightgray;
+      }
+
+      .panels {
+        padding: 8px;
+        border: 1px solid #ccc;
+        border-radius: 4px;
+        margin-top: 8px;
+      }
+
+      .panel {
+        display: none;
+      }
+
+      .panel.active {
+        display: block;
+      }
+
+      .dots {
+        display: flex;
+        justify-content: flex-start;
+        margin-top: 4px;
+        padding-bottom: 16px;
+      }
+
+      .dot {
+        margin: 0 4px;
+        border-radius: 50%;
+        border: 1px solid #ccc;
+        width: 20px;
+        height: 20px;
+      }
+
+      .dot:last-child {
+        margin-right: 0;
+      }
+
+      .dot.active {
+        background-color: blue;
+        border-color: blue;
+      }
+
+      .my-modified-toolbar-section {
+        position: absolute;
+        right: 0;
+        bottom: 0;
+      }
+
+      [dir="rtl"] .my-modified-toolbar-section {
+        right: auto;
+        left: 0;
+      }
+
+    </style>
+  </head>
+  <body class="mdc-typography">
+    <div class="demo-body">
+
+      <h1>Tab Bar</h1>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-rtl">Toggle RTL</button>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-dark-theme">Toggle Dark Theme</button>
+
+      <section>
+        <h2>Basic Tab Bar</h2>
+        <nav id="basic-tab-bar" class="mdc-tab-bar">
+          <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+          <a class="mdc-tab" href="#two">Item Two</a>
+          <a class="mdc-tab" href="#three">Three</a>
+          <span class="mdc-tab-bar__indicator"></span>
+        </nav>
+      </section>
+
+      <section>
+        <h2>CSS-Only Tab Bar</h2>
+        <nav class="mdc-tab-bar">
+          <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+          <a class="mdc-tab" href="#two">Item Two</a>
+          <a class="mdc-tab" href="#three">Three</a>
+          <span class="mdc-tab-bar__indicator"></span>
+        </nav>
+      </section>
+
+      <section>
+        <h2>Icon Tab Labels</h2>
+        <nav id="icon-tab-bar" class="mdc-tab-bar mdc-tab-bar--icon-tabs">
+          <a class="mdc-tab mdc-tab--active" href="#recents">
+            <i class="material-icons mdc-tab__icon" aria-label="Recents">phone</i>
+          </a>
+          <a class="mdc-tab" href="#favorites">
+            <i class="material-icons mdc-tab__icon" aria-label="Favorites">favorite</i>
+          </a>
+          <a class="mdc-tab" href="#nearby">
+            <i class="material-icons mdc-tab__icon" aria-label="nearby">person_pin</i>
+          </a>
+          <span class="mdc-tab-bar__indicator"></span>
+        </nav>
+      </section>
+
+      <section>
+        <h2>CSS-Only Icon Labels</h2>
```

Comment:
That sounds fine. What about keeping CSS-Only? I think it implies that the best experience will be with the fully upgraded JavaScript component, and that sits well with me.

"CSS-Only Icon Tab Labels" ?